lib/execution_engine2/sdk/job_submission_parameters.py

```diff
@@ -124,9 +124,11 @@ def check_parameters(
             client_group,
             None if client_group_regex is None else bool(client_group_regex),
             _check_string(bill_to_user, "bill_to_user", optional=True),
-            bool(ignore_concurrency_limits),
+            None
+            if ignore_concurrency_limits is None
+            else bool(ignore_concurrency_limits),
```

Comment:
To me the previous 3 lines are not readable at all, but black likes it that way

Reply:
?!?! With this formatting, I would not have guessed that this is python's equivalent of a ternary. Thanks black!